packages/ui/src/components/Tabs/utils.ts

```diff
@@ -38,7 +38,7 @@ export const commonTabStyles = {
     `,
     "data-h2-color": `
       base(black)
-      base:selectors[[data-state='active']](primary.darker)
+      base:selectors[[data-state='active']](black.darker)
```

Comment:
I believe this was supposed to be applied to [the focused style only](https://github.com/GCTC-NTGC/gc-digital-talent/issues/8785#:~:text=Focused%20tabs%20should,black%20font%20color) (not active).  This should probably be:

```tsx
      base(black)
      base:focus-visible(black)
      base:selectors[[data-state='active']](primary.darker)
      base:selectors[[data-state='active']]:focus-visible(black)
```

Reply:
@esizer Yeah, you've got it. The black text and blue focus color should be applied only on `focus-visible` but applies to both the generic and active states. The difference between them is that the active state, when focused, retains the bolding.